loops-and-arrays.js

```diff
@@ -12,8 +12,11 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
-
-
+console.log(names)
+for (let i = 0; i < names.length; i++) {
+    names[i] = names[i] + 'je'
```

Comment:
Dat je achter de regels geen ";" gebruikt zal een bewuste keuze zijn(?) aangezien je dat nergens gebruikt en het ook niet noodzakelijk in JS.